opal/include/opal/sys/sync_builtin/atomic.h

```diff
@@ -45,7 +45,16 @@ static inline void opal_atomic_wmb(void)
     __sync_synchronize();
 }
 
+// This avoids a duplicate definition in PMIx <v1.2.5 (Open MPI v2.1.x
+// has PMIx 1.2.4).  This was fixed in PMIx v2.1.5, but it is not
```

Comment:
it was fixed in PMIx v1.2.5  <-- you typo'd the version

Reply:
Thx -- fixed.